app/src/main/java/cm/aptoide/pt/analytics/FirstLaunchAnalytics.java

```diff
@@ -74,6 +87,88 @@ public void sendFirstLaunchEvent(String utmSource, String utmMedium, String utmC
         FIRST_LAUNCH, AnalyticsManager.Action.OPEN, "");
   }
 
+  private void sendPlayProtectEvent() {
+    SafetyNet.getClient(context)
```

Comment:
Like I said above, i would inject the safety net client on the class instead of getting it here using the context